### PR TITLE
Feat: Show Installed apps

### PR DESCRIPTION
**Pull request by ignaciosantise**
### Summary
- Showing installed wallets for Android & iOS
- Added a native module to detect wallets installed in Android
- Solved an issue with FlatList and babel imports

#### iOS

https://user-images.githubusercontent.com/25931366/226438705-3f03215d-5241-48f5-beec-351af8ccbe1e.MP4

#### Android

https://user-images.githubusercontent.com/25931366/226438733-74051e71-bd87-4ccc-b5da-67f9e4f158d6.mp4

### CODE REVIEW

File: dapps/v2Explorer/babel.config.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 module.exports = {
   presets: ['module:metro-react-native-babel-preset'],
   plugins: [
+    ['@babel/plugin-transform-flow-strip-types', {loose: true}], //https://github.com/facebook/react-native/issues/29084#issuecomment-1030732709
```
Comment (crypblizz8) [inline dapps/v2Explorer/babel.config.js:4]: 👍 Thanks for linking + Can remove the comment

File: dapps/v2Explorer/src/components/ExplorerModal.tsx [hunk 2]
```diff
@@ -4,14 +4,14 @@ import Modal from 'react-native-modal';
 import {InitialExplorerContent} from './InitialExplorerContent';
 import {ViewAllExplorerContent} from './ViewAllExplorerContent';
 
-import {fetchInitialWallets, fetchViewAllWallets} from '../utils/ExplorerUtils';
```
Comment (crypblizz8) [inline dapps/v2Explorer/src/components/ExplorerModal.tsx:7]: 🙏

File: dapps/v2Explorer/ios/v2Explorer/Info.plist [hunk 3]
```diff
@@ -22,6 +22,15 @@
 	<string>????</string>
 	<key>CFBundleVersion</key>
 	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<key>LSApplicationQueriesSchemes</key>
```
Comment (crypblizz8) [inline dapps/v2Explorer/ios/v2Explorer/Info.plist:25]: I understand I left these in and forgot to remove them. So are we only showing 6 specific wallets or has that not been decided yet?

File: dapps/v2Explorer/android/app/src/main/java/com/v2explorer/MainApplication.java [hunk 4]
```diff
@@ -24,7 +24,7 @@ protected List<ReactPackage> getPackages() {
           @SuppressWarnings("UnnecessaryLocalVariable")
           List<ReactPackage> packages = new PackageList(this).getPackages();
           // Packages that cannot be autolinked yet can be added manually here, for example:
-          // packages.add(new MyReactNativePackage());
+           packages.add(new CustomPackage());
```
Comment (crypblizz8) [inline dapps/v2Explorer/android/app/src/main/java/com/v2explorer/MainApplication.java:27]: NonBlocking / Curious: Never added customPackages myself. Do you have any references I can read / look at for future?

**Comment by crypblizz8** [comment]
Looking awesome. Reviewed the QRCode + Installed Wallet Parts. After we figure out what wallets to `hardcode` for iOS we can approve and move onto the next step!

File: dapps/v2Explorer/android/app/src/main/java/com/v2explorer/MainApplication.java (hunk 4 above)
Comment (ignaciosantise) [reply dapps/v2Explorer/android/app/src/main/java/com/v2explorer/MainApplication.java:27]: Yep! Here you have the links:

Android guide: https://reactnative.dev/docs/native-modules-android
iOS guide: https://reactnative.dev/docs/native-modules-ios

When we move all into an SDK, i'll try to migrate the code to Kotlin

File: dapps/v2Explorer/ios/v2Explorer/Info.plist (hunk 3 above)
Comment (ignaciosantise) [reply dapps/v2Explorer/ios/v2Explorer/Info.plist:25]: mm i'll add em all, and think an easier way to do it when we migrate to an sdk

**Pull request merged**